tests/integration/objectives_plot/test_objectives_plot_integration.py

```diff
@@ -5,14 +5,14 @@
 
 def test_objective_plot_callback(app, dash_duo, mocker, caplog):
     test_data = [
-        {"batch": 0, "realization": 1, "function": "f0", "value": 100, "simulation": 0},
+        {"batch": 0, "realization": 1, "function": "f0", "value": 100, "simulation": 1},
```

Comment:
A very nitpicky comment in real cases we cannot have the same simulation number twice for one realization.

Reply:
of course, good catch!

Reply:
On second thought, maybe not. These are two function values f0 and f1 that are produced in the same job in the same realization, that is possible right>

Reply:
Does not matter if there are two functions that is equivalent to a multi-objective optimization case. For one geo-realization simulation numbers will not repeat. You can try to verify this by running the `config_multiobj.yml` config from the math_func examples.

Reply:
Indeed, it would be a multi-objective case, that is where you get this situation. Running  `config_multiobj.yml` gives the following for batch 0 from the `api.objective_values`:
```
{'function': 'distance_p', 'batch': 0, 'realization': '0', 'simulation': '0', 'value': -0.75}
{'function': 'distance_q', 'batch': 0, 'realization': '0', 'simulation': '0', 'value': -4.75},
```
So, `batch=0`, `realization=0` and `simulation=0` has two objective values originating from a single simulation run. Indeed there is one simulation folder `simulation_0`, which has two result files: `distance_p_0` and `distance_q_0`, which corresponds to these two lines.

Reply:
You are right! I got a bit confused with the simulation numbers for the different objective functions